packages/console/src/hooks/use-swr-fetcher.ts

```diff
@@ -1,21 +1,26 @@
-import { RequestErrorMetadata } from '@logto/schemas';
+import { RequestErrorBody } from '@logto/schemas';
+import { HTTPError } from 'ky';
 import { useCallback } from 'react';
 import { BareFetcher } from 'swr';
 
 import useApi, { RequestError } from './use-api';
 
 const useSwrFetcher = () => {
-  const api = useApi();
+  const api = useApi({ hideErrorToast: true });
```

Comment:
Why is `hideErrorToast` always `true`?

Reply:
When using `useSWR`, if error occured, it will keep tring, and then the error toast will keep showing again and again. Is a best practice to handle error manully, that's why we use it like:
```
const { data, error } = useSWR('/api');
```

Reply:
ok